tutor/resources/styles/components/modals.scss

```diff
@@ -1,6 +1,9 @@
+$modal-header-height: 75px;
+$modal-header-font-size: 16px;
+
 .modal {
-  $header-padding: 35px;
-  $modal-padding: 50px;
+  $header-padding: ($modal-header-height - $modal-header-font-size) / 2;
+  $modal-padding: 30px;
 
```

Comment:
@pandafulmanda have you ran this by @Fredasaurus and @rvnewell ?  I'd done this earlier and they didn't want it changed globally.  In particular they didn't like how the enrollment modal looked after this change

Reply:
ah, okay, i can make it local to the enrollment modal